packages/yoga/src/Input/web/Field.jsx

```diff
@@ -74,8 +74,11 @@ const Field = styled.input`
 
         font-weight: ${input.label.font.weight};
         color: ${
-          error ? `${colors.feedback.attention[1]}` : `${colors.text.primary}`
+          error || (value && error)
```

Comment:
I think that this expression can be simplified to only: `error`. Like it was before.

1. If `error` value is **truthy**, the expression is **truthy** on the first condition.
2. if `error` is value is **falsy**, both `error` and `value && error` will be **falsy**

Reply:
Thanks for the push, I'm moving this condition to the right place.